src/sedml/SedErrorLog.cpp

```diff
@@ -88,7 +88,7 @@ SedErrorLog& SedErrorLog::operator=(const SedErrorLog& other)
 /*
  * Used by the Destructor to delete each item in mErrors.
  */
-struct Delete
+struct Delete : public unary_function<XMLError*, void>
```

Comment:
this will not compile with newer c++ versions, as unary_function got removed, but i can sort that